static_frame/test/unit/test_index_base.py

```diff
@@ -7,42 +7,7 @@
 
 
 class TestUnit(TestCase):
-    def test_index_base_slotted_a(self) -> None:
-        idx1 = IndexBase()
-
-        with self.assertRaises(AttributeError):
-            idx1.g = 30  # type: ignore
-        with self.assertRaises(AttributeError):
-            idx1.__dict__
-
-    def test_index_base_not_implemented(self) -> None:
-        idx1 = IndexBase()
-
-        with self.assertRaises(NotImplementedError):
-            idx1._ufunc_axis_skipna(
-                axis=0,
-                skipna=False,
-                ufunc=np.sum,
-                ufunc_skipna=np.nansum,
-                composable=True,
-                dtypes=(),
-                size_one_unity=True,
-            )
-
-        with self.assertRaises(NotImplementedError):
-            idx1._update_array_cache()
-
-        with self.assertRaises(NotImplementedError):
-            idx1.copy()
-
-        with self.assertRaises(NotImplementedError):
-            idx1.copy()
-
-        with self.assertRaises(NotImplementedError):
-            idx1.display()
-
-        with self.assertRaises(NotImplementedError):
-            idx1.from_labels(())
+    pass
```

Comment:
This is no longer possible given that `IndexBase` is now an incomplete ABC class, and so you can't even instantiate `IndexBase()`